### PR TITLE
4.1.0rc2

### PR DESCRIPTION
**Pull request by poodlewars**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -129,6 +129,5 @@ extra:
     - willdealtry
     - poodlewars
     - vasil-pashov
-    - GemDot
```
Comment (poodlewars) [inline recipe/meta.yaml:132]: I think Gem's username has changed, so the linting insisted on this change.

**Comment by poodlewars** [comment]
@conda-forge-admin, please rerender

**Review by jjerphan — APPROVED** [review]
LGTM.

---

Side-comment unrelated to this PR: `rc` and `main` must ideally be identical modulo the `libevent` pinning and the `version`. Currently (i.e. before the merge of this PR) they are not.

To do so, we need to integrate [some of the absolute changes in between `rc` and `main`](https://github.com/conda-forge/arcticdb-feedstock/compare/rc..main) which comes from conda-forge migrations. Namely we need to:
 - integrate the migration in `.ci_support/migrations`:
   - https://github.com/conda-forge/arcticdb-feedstock/compare/rc..main#diff-c383770dd0fcb01b355785cbf4ec54892d0e3c13e62cb123e15d0ce3940ce5a7 
   - https://github.com/conda-forge/arcticdb-feedstock/compare/rc..main#diff-71ad53b6d852a8c28eed0f9647f6017b39901f170a020e965642738686839f33
 - integrate some of [the elements in the recipe for supporting libprotobuf 4](https://github.com/conda-forge/arcticdb-feedstock/compare/rc..main#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9a)
 - re-render the recipe to generate all the other files for the CI.

We need to find a way to maintain this consistency (e.g. really backport changes as soon as possible or see if the bot can be configured) without much maintenance cost if possible.

I can do that tomorrow.

**Comment by jjerphan** [comment]
Done with:
 - https://github.com/conda-forge/arcticdb-feedstock/pull/98
 - https://github.com/conda-forge/arcticdb-feedstock/pull/99
 - https://github.com/conda-forge/arcticdb-feedstock/pull/100
 - https://github.com/conda-forge/arcticdb-feedstock/pull/101

**Pull request merged**